com.jagex.RuneScape.json

```diff
@@ -0,0 +1,148 @@
+{
+    "app-id": "com.jagex.RuneScape",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "runescape",
+    "separate-locales": false,
+    "tags": ["proprietary"],
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Sound access */
+        "--socket=pulseaudio",
+        /* Network access */
+        "--share=network",
+        /* OpenGL */
+        "--device=dri",
+        "--persist=Jagex",
+        "--extra-data=runescape.deb:0265f6c12711e90299cb3793e9dab45a9de6194f7400d3c7c855c176f209b2b8:3014960::https://content.runescape.com/downloads/ubuntu/pool/non-free/r/runescape-launcher/runescape-launcher_2.2.4_amd64.deb"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/lib/libXmuu*",
+        "/lib/libXmu.so",
```

Comment:
These are just symlinks right? Can just leave em be rather than complicating the manifest.